Generic-worker intermittent test failure

Happens few times for the `generic-worker-build` tests:
https://community-tc.services.mozilla.com/tasks/XpYtzedNQrCLhj_68swXgg/runs/0/logs/live/public/logs/live.log

```
2024/06/03 10:43:25 Started taskcluster proxy process (PID 11089)
2024/06/03 10:43:25 Version: Taskcluster proxy 65.1.0
2024/06/03 10:43:25 Listening on: 127.0.0.1:34569
2024/06/03 10:43:25 Root URL: 'https://community-tc.services.mozilla.com'
2024/06/03 10:43:25 Client ID: 'project/taskcluster/generic-worker/taskcluster-ci'
2024/06/03 10:43:25 Access Token: <not shown>
2024/06/03 10:43:25 Warning - no taskcluster certificate set - assuming permanent credentials are being used
2024/06/03 10:43:25 Proxy with scopes:  [queue:get-artifact:SampleArtifacts/_/X.txt]
2024/06/03 10:43:25 listen tcp 127.0.0.1:34569: bind: address already in use
2024/06/03 10:43:25 Stopped taskcluster proxy process (PID 11089)
--- FAIL: TestTcProxy (0.01s)
    tcproxy_test.go:55: Could not interpret response "URL /api/auth/v1/scopes/current with method GET NOT FOUND\n" as json: invalid character 'U' looking for beginning of value
FAIL
FAIL	github.com/taskcluster/taskcluster/v65/workers/generic-worker/tcproxy	0.025s
ok  	github.com/taskcluster/taskcluster/v65/workers/generic-worker/win32	1.011s
FAIL
```

Some race condition around:
```
2024/06/03 10:43:25 Listening on: 127.0.0.1:34569
..
2024/06/03 10:43:25 listen tcp 127.0.0.1:34569: bind: address already in use
```

--- a/changelog/ajz5NZzdQrS0XJd3QNy24g.md
+++ b/changelog/ajz5NZzdQrS0XJd3QNy24g.md
@@ -1,0 +1,3 @@
+audience: general
+level: silent
+---

--- a/workers/generic-worker/tcproxy/tcproxy_test.go
+++ b/workers/generic-worker/tcproxy/tcproxy_test.go
@@ -28,7 +28,7 @@ func TestTcProxy(t *testing.T) {
 		Certificate:      certificate,
 		AuthorizedScopes: []string{"queue:get-artifact:SampleArtifacts/_/X.txt"},
 	}
-	ll, err := New(executable, 34569, rootURL, creds)
+	ll, err := New(executable, 34570, rootURL, creds)
 	// Do defer before checking err since err could be a different error and
 	// process may have already started up.
 	defer func() {
@@ -40,7 +40,7 @@ func TestTcProxy(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not initiate taskcluster-proxy process:\n%s", err)
 	}
-	res, err := http.Get("http://localhost:34569/auth/v1/scopes/current")
+	res, err := http.Get("http://localhost:34570/auth/v1/scopes/current")
 	if err != nil {
 		t.Fatalf("Could not hit url to download artifact using taskcluster-proxy: %v", err)
 	}

